src/common/transformations/include/transformations/rt_info/nms_selected_indices.hpp

```diff
@@ -21,7 +21,7 @@ TRANSFORMATIONS_API bool has_nms_selected_indices(const Node* node);
 
 TRANSFORMATIONS_API void set_nms_selected_indices(Node* node);
 
-class TRANSFORMATIONS_API NmsSelectedIndices : ov::RuntimeAttribute {
+class TRANSFORMATIONS_API NmsSelectedIndices : public ov::RuntimeAttribute {
```

Comment:
How is it related to build with C++20?
Why it was not required to be public before?